crates/rgkl/.cargo/config.toml

```diff
@@ -0,0 +1,6 @@
+[target.aarch64-unknown-linux-gnu]
+linker = "aarch64-linux-gnu-gcc"
+
+[profile.release]
+incremental = true
+debug = true
```

Comment:
Doesn't setting `debug = true` here change the setting for all "release" profiles? We also use the "release" profile when we build for production: https://github.com/kubetail-org/kubetail/blob/main/build/package/Dockerfile.cluster-agent#L32

Instead, can we specifically tell cargo to use the debug profile in development? Would something like this work?
```console
 cargo build --target aarch64-unknown-linux-gnu --debug
 ```